mantidproject/mantid
# VSI: keep the state of views when switching between them

This fixes [#11708](http://trac.mantidproject.org/mantid/ticket/11708).

**To test**:
- check code
- check that it works. The state (visual state) of each view should be restored when you switch back to it after using a different view. This should work for the 4 types of views. You can reset the state of the views pushing a new button added here, just right of the color selection widget. 

The approach taken does the job but there's an issue that should/will be addressed in [#11739](http://trac.mantidproject.org/mantid/ticket/11739)
